app/models/miq_worker.rb

```diff
@@ -273,17 +252,11 @@ def self.start_worker(*params)
 
   cache_with_timeout(:my_worker) { server_scope.find_by(:pid => Process.pid) }
 
-  def self.find_all_current(server_id = nil)
-    MiqWorker.find_current(server_id)
-  end
 
-  def self.stop_all_workers(server_id = nil)
-    MiqWorker.stop_workers(server_id)
+  def self.stop_all_workers
+    MiqWorker.stop_workers
```

Comment:
Could this be an alias instead? Or just delete this method and change the one caller in `MiqWorker.atShutdown`?